src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -704,8 +704,8 @@ private function addPhpErrorsSection(ArrayNodeDefinition $rootNode)
                     ->children()
                         ->booleanNode('log')
                             ->info('Use the app logger instead of the PHP logger for logging PHP errors.')
-                            ->defaultValue(false)
-                            ->treatNullLike(false)
+                            ->defaultValue($this->debug)
+                            ->treatNullLike($this->debug)
```

Comment:
This config is just how things work since 2.7 at least (and before I guess): in debug mode, we log deprecations and silenced errors to show them in the profiler. In prod (aka !debug), the current strategy is to let the native PHP error handler to deal with any error. This was hidden hard coded in the DI extension and is now exposed to users. There is no change here from what's done since years...